app/src/components/report/LocationPicker.tsx

```diff
@@ -0,0 +1,222 @@
+        >
+            <Map
+                mapboxAccessToken={MAPBOX_ACCESS_TOKEN}
+                reuseMaps={true} //remove this if map breaks
+                initialViewState={{ ...viewState }}
+                style={{ width: '100%', height: '100%', borderRadius: 5 }}
+                mapStyle="mapbox://styles/mapbox/streets-v9"
+                onClick={() => {
+                    if (draggableMarker) return;
+                    modalRef.current?.openModal();
+                }}
+            >
+                {!isUndefined(reportCoords) && (
+                    <Marker
+                        longitude={reportCoords!.longitude}
+                        latitude={reportCoords!.latitude}
+                        color="red"
+                        draggable={draggableMarker}
+                        onDragEnd={handleDragEnd}
+                    />
+                )}
+                <LocationSearchBar
+                    placeMarker={updateReportCoords}
+                    proximity={
+                        !isUndefined(reportCoords)
+                            ? {
+                                  longitude: reportCoords!.longitude,
+                                  latitude: reportCoords!.latitude,
+                              }
+                            : undefined
+                    }
+                />
+                <GeolocateControl
+                    position="bottom-right"
+                    onGeolocate={handleGeolocate}
+                    onError={(error) => {
+                        window.alert(error.message);
+                    }}
+                />
+            </Map>
+            <MyModal
+                ref={modalRef}
+                header="Alert!"
+                description="Do you want to change the location? This will reset the current address."
+                denyMessage="No, keep the current location"
+                confirmMessage="Yes, I want to update the location"
+                onConfirm={resetLocationAndAddress}
+            />
+            <div
+                style={{
+                    marginTop: 10,
+                    textAlign: 'center',
+                }}
+            >
+                <Button disabled={locationSubmitted} onClick={reverseGeolocation}>
+                    {!locationSubmitted ? 'Report at this location' : 'Location already submitted'}
+                </Button>
+            </div>
+
+            {/* <Form>
+                <Form.Group>
+                    <Form.Control placeholder="Coordinates" value={`${reportCoords?.longitude || ''}, ${reportCoords?.latitude || ''}`} />
+                </Form.Group>
+                <Form.Group>
+                    <Form.Control placeholder="Address" value={reportAddress} />
+                </Form.Group>
+                <Button onClick={reverseGeolocation}>Reverse Geo</Button>
+                <Button onClick={() => submitLocation(reportCoords!)}>Submit Location</Button>
+            </Form> */}
```

Comment:
Is this necessary? If not, maybe we can remove it?

It's better not to leave commented-out code since it can be hard to tell the purpose (is it broken? does it work? is it old and outdated? is it the future?)

Reply:
Yes I'll remove it. It's for testing purposes.